### ISSUE TITLE
coil:configure returning invalid username and password

### ISSUE THREAD

**Issue by SamuelEVicente** [sharafian/web-monetization-access#2]
When using coil:configure with moneyd it returns invalid user and password, But I double checked my user and pass and its correct, I was wondering if there is any more configuration I need to do with coil before running moneyd coil:configure?

**Comment by SamuelEVicente** [comment]
Hi @sharafian , I am having trouble with configuring coil so I can be able to manipulate / proof the payment using the server. I'm using the proper credentials that it ask's me for but it fails. Is there anything that I might be missing?

**Comment by matdehaast** [comment]
Hey @SamuelEVicente 

Can you explain your use case a bit more. Monetization from Coil terminates at the wallet of your Payment Pointer not Coil.

**Comment by sharafian** [comment]
Yeah, moneyd uplink coil is experimental and shouldn't be used for any production use cases. If you want to verify payments on the server side then I recommend using STREAM receipts when they're available (which will be soon) along with a receipt verifier instance (https://github.com/coilhq/receipt-verifier).

This repo is mainly a starting point for sites that have a interledger peering relationship, which usually requires some kind of peering agreement with a wallet and would typically be done by businesses

**Comment by SamuelEVicente** [comment]
Ok, Thank you @sharafian for responding but I'm a little confused. If I have a wallet setup with xrptipbiot and have a payment pointer, and it is currently being used by the client side, how exactly am I able to verify the payments to make sure users don't spoof the monetization events? With my understanding of receipt-verifier I would need to submit a STREAM Receipt to the verifier but how would I get the STREAM receipts from my wallet/payment pointer? I apologize for my ignorance about this. Thank you again! :)

**Comment by SamuelEVicente** [comment]
@matdehaast Sorry for the delay, My use case would be verifying the payment with the server to make sure that the user is not spoofing the monetization events with the bookmarklet technique used for testing.

**Comment by sharafian** [comment]
Unfortunately the STREAM receipts feature isn't available just yet (wallets are rolling it out soon, though). But when it's been implemented on the wallet side it will be emitted from Web Monetization's `monetizationprogress` events in

**Comment by SamuelEVicente** [comment]
@sharafian  Ok so right now there is no valid way to proof a payment for the production environment? The only way to simulate it is by using the testing environment with web-monetization-access?

**Comment by sharafian** [comment]
yeah, in general there isn't a way to verify payments in production. Some sites which have configured Interledger access by setting up peering relationships are verifying payments in a production environment (coil.com, cinnamon.video) but in those peering relationships require business agreements and aren't easily available to individuals.

The hope with receipts is to expand it so that anybody can validate payments without needing to peer to the Interledger network directly
